Observer/TescoPaymentObserver.php

```diff
@@ -39,19 +39,15 @@ public function execute(Observer $observer)
     {
         $order   = $observer->getEvent()->getOrder();
         $payment = $order->getPayment();
-        
-        if (!$payment) {
```

Comment:
this is from _Payment observer_ I don't expect $payment be ever null.

Reply:
well actually it is trigged after `sales_order_place_after` event.. still don't expect it can be `null`